frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -440,6 +511,25 @@ def wires(self):
         return total_wires
 
 
+# TODO: Should we inherit from AdjointOperation? Maybe better to mixin the different behaviours like
+# PennyLane does (AdjointOperation, AdjointObs, AdjointOperationObs).
+class Adjoint(AdjointOperation, HybridAdjoint):
+    """This class provides near identical behaviour as PennyLane for adjoint instances with only a
+    single base operation. Additionally, it provides the same functionality as HybridAdjoint."""
+
+    def __init__(self, base_op, tracing_artifacts):
+        AdjointOperation.__init__(self, base_op)
```

Comment:
Can we use the `Adjoint.decomposition` method now?

Reply:
For single ops I believe so

Reply:
I tested this locally and we now use `Adjoint.decomposition` with single ops :partying_face: